src/chronos/df_utils.py

```diff
@@ -4,6 +4,7 @@
 # Authors: Abdul Fatir Ansari <ansarnd@amazon.com>
 
 
+import warnings
```

Comment:
@abdulfatir Regarding the accuracy test for `predict_df`, maybe we can just benchmark Chronos-2 within AutoGluon and make sure that the scores match the standalone model? Otherwise we could extend the unit tests to make sure that the outputs match exactly between the different APIs